nuxeo-features/nuxeo-platform-mail/nuxeo-platform-mail-core/src/main/java/org/nuxeo/ecm/platform/mail/utils/MailCoreHelper.java

```diff
@@ -134,15 +133,14 @@ protected static void doCheckMail(DocumentModel currentMailFolder, CoreSession c
 
             initialExecutionContext.put(CORE_SESSION_KEY, coreSession);
 
-            initialExecutionContext.put(LEAVE_ON_SERVER_KEY, Boolean.TRUE); // TODO should be an attribute in 'protocol'
-                                                                            // schema
+            // TODO should be an attribute in 'protocol' schema
+            initialExecutionContext.put(LEAVE_ON_SERVER_KEY, Boolean.TRUE);
```

Comment:
perhaps just a static import like for the `FALSE.toString` once or use directly `true` ;)

Reply:
Why using `toString`? previous code already gave a Boolean 🤔

Reply:
I changed to import statically TRUE.